composer.json

```diff
@@ -18,6 +18,7 @@
     }
   },
   "require": {
-    "aws/aws-sdk-php": "3.x"
+    "aws/aws-sdk-php": "3.x",
+    "symfony/console": "^2.6||^4.0"
```

Comment:
Seems like there's a lot of Magento framework & module libraries that are used, should we not list them as well here?

Reply:
Added